src/openforms/submissions/pricing.py

```diff
@@ -76,6 +77,12 @@ def get_submission_price(submission: Submission) -> Decimal:
     data = submission.data
     # test the rules one by one, if relevant
     price_rules = form.formpricelogic_set.all()
+    if price_rules:
+        warnings.warn(
+            "Price logic rules are no longer supported. The left-over implementation "
```

Comment:
Yes, that's implied! I can't delete the models yet because then this implementation breaks and that breaks the tests. I'll make sure to get rid of that once 3.0 is released.